src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -31,6 +33,8 @@ export const MapRoom: React.FC<MapRoomProps> = ({
   const hasRecentRoomUsers = recentRoomUsers.length > 0;
 
   const isUnclickable = room.type === RoomTypes.unclickable;
+  const isIframe = room.type === RoomTypes.iframe;
+  const isCovertRoom = room.type && COVERT_ROOM_TYPES.includes(room.type);
```

Comment:
The only thing that concerns me is the naming here, but it's definitely NB

Reply:
As I mentioned in the other comment thread (can’t get link as on phone atm), I believe there was a slack discussion + decisions made around naming somewhere.

Reply:
In fact, it's the thread linked from https://github.com/sparkletown/sparkle/pull/1427#discussion_r638451622